queue_services/account-mailer/src/account_mailer/email_templates/creditcard_refund_request_email.html

```diff
@@ -1,72 +1,13 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <base href="/">
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="referrer" content="origin-when-cross-origin"/>
-    <meta name="author" content="BC Registries and Online Services">
-    <title>Refund Request for {{ refund_data.identifier }}</title>
```

Comment:
We lost this information

Reply:
The email subject has this information. 

https://github.com/bcgov/sbc-auth/blob/main/queue_services/account-mailer/src/account_mailer/email_processors/refund_requested.py#L37